packages/vkui/src/components/PanelHeaderButton/PanelHeaderButton.tsx

```diff
@@ -17,12 +17,23 @@ const platformClassNames = {
 };
 
 export interface PanelHeaderButtonProps extends Omit<TappableProps, 'label'> {
+  /**
+   * Флаг для обозначения первичной кнопки
+   * Влияет на стилизацию кнопки.
+   */
   primary?: boolean;
-  // TODO [>=8]: добавить св-во `indicator`, чтобы разграничить кейсы
+  /**
+   * Текст или содержимое кнопки
+   * TODO [>=8]: добавить св-во indicator, чтобы разграничить кейсы.
+   */
   label?: React.ReactNode;
```

Comment:
```suggestion
  /**
   * Текст или содержимое кнопки
   */
  label?: React.ReactNode; // TODO [>=8]: добавить св-во indicator, чтобы разграничить кейсы.
```
чтобы не попала на страницу доки